Makefile

```diff
@@ -1,18 +1,190 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Looks like the Makefile in the root got overwritten by cmake. The cmake builds should be run in `./build`. This is done automatically by running `make build` in the root directory.